recipe/conda_build_config.yaml

```diff
@@ -11,5 +11,5 @@ libclang_soversion:
 
 variant:
   - default
-  - root_63402  # [linux or osx]
+  - root_63406  # [linux or osx]
```

Comment:
So given that nothing about the patches has changed, I presume this is just to have consistent builds for `_63406` across llvmdev and clangdev?

Reply:
Yes exactly, I believe the consistency is nice to have. 👍